src/inputgroup/index.css

```diff
@@ -63,3 +74,77 @@
     right: 0;
   }
 }
+
+.spectrum-Datepicker--range {
+  border-radius: var(--spectrum-border-radius);
+  /* Input Group */
+  &.spectrum-InputGroup--quiet {
+    border-radius: var(--spectrum-combobox-quiet-fieldbutton-border-radius);
+    .spectrum-FieldButton {
+      border-radius: var(--spectrum-combobox-quiet-fieldbutton-border-radius);
+    }
+  }
+  /** Datetime Range **/
+  &.spectrum-Datepicker--datetimeRange {
+    /* Inputs */
+    .spectrum-InputGroup-field {
+      width: var(--spectrum-datepicker-datetime-input-width);
+      min-width: var(--spectrum-datepicker-datetime-input-width);
+    }
+  }
+  /* Inputs */
+  .spectrum-InputGroup-field {
+    width: var(--spectrum-datepicker-input-width);
+    min-width: var(--spectrum-datepicker-input-width);
+    &:nth-of-type(1) {
+      border-right: 0;
+      padding-right: var(--spectrum-padding);
+      &.is-invalid,
+      &:invalid {
+        background-image: none;
+        padding-right: var(--spectrum-padding);
+      }
+    }
+    &:nth-of-type(2) {
```

Comment:
@GarthDB how does this make you feel? Do you think we should have selectors for these two bad boys instead of doing `nth-of-type`?

Reply:
It has full browser support, so I don't really have any issue with it. https://caniuse.com/#feat=css-sel3

Reply:
The only other reason we might be concerned is performance and/or code readability/maintainability, but I think it's fine on both of those.

Reply:
I think the specificity is pretty high here. I'm not 100% against it, but it feels like we could throw a class on  the inputs and simplify the CSS... The real question is: do we want markup simplicity (the PR, as-is), or CSS simplicity (throw an extra class on the fields)?

Reply:
@majornista We talked a bit with @devongovett and think these selectors should be classes added to the markup.

Reply:
@lazd Would `spectrum-Datepicker-field--startDatetime` and `spectrum-Datepicker-field--endDatetime` work?